proceeding/text/data.tex

```diff
@@ -31,4 +29,12 @@ \section{Data}
   \end{itemize}
 \end{enumerate}
 
-(Table of number of sources per catalog (and maybe also number of sources in each class?))
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/four_images}}
+  \caption{Survey images (left to right): Fermi color, Akari 90um, Planck LFI, Planck HFI.}
```

Comment:
Maybe also put the label of which sphere is which on the image? Then it can easily be copied onto posters and slides and is quicker to grok.